README.md

```diff
@@ -1,168 +1,123 @@
-
-[nvEncConfig.json](https://github.com/CatalystCode/3dtoolkit/blob/v0.1.0/Plugins/NativeServerPlugin/nvEncConfig.json) is the nvencode configuration file.  
-+ Set "useSoftwareEncoding" to true to use the CPU for video encoding - this will increase latency and should only be used for development on computers without an nvidia video card.
-+ Set "serverFrameCaptureFPS": 60 and "fps" to the same value.  Maximum FPS is dependent on card and scene complexity.
-+ Set "bitrate" and "minBitrate" to set the max and min bitrates for video streaming - recommended maximum @ 10mbps and min to 5.5mbps for high quality streaming.  Anything over 10mbps will not visibly increase quality (see test runners for validation).  Setting minBitrate to 0 enables webrtc to drop bitrate to whatever it needs to in order to keep video streaming fluidly.
-
-## Distributing binaries
-
-> Note: Currently for others you'll need to zip and upload yourself to match the `.\WebRTCLibs\webrtcInstallLibs.ps1` script.
+## Build output
 
-https://3dtoolkitstorage.blob.core.windows.net/libs/m58patch_headers.zip
-https://3dtoolkitstorage.blob.core.windows.net/libs/m58patch_Win32.zip
-https://3dtoolkitstorage.blob.core.windows.net/libs/m58patch_x64.zip
+After you've built the solution, you'll likely want to start one sample server implementation, and one sample client implementation. These are applications that we've build to demonstrate the behaviors the toolkit provides.
 
-## Standing up production TURN/STUN/Signaling Servers
+> Note: We advise you to try `Spinning Cube` and `Win32Client` to begin, as these are the simpliest sample implementations.
 
-+ Signaling Server (Node.JS) - https://github.com/anastasiia-zolochevska/signaling-server
-+ TURN Server (optional, needed for VPN/Proxy networks) - https://github.com/anastasiia-zolochevska/coturn-to-azure-deployment
+Here's a table illustrating where each sample implementation will be built to. To run one server and one client, navigate to that location and start the `exe`.
 
-## Building WebRTC Libraries from Source (You don't need to do this unless you are changing the encoder)
+> Note: the following table describes the location under `Build\<Platform>\<Configuration>\` where a sample can be found. To identify what `<Platform>` and `<Configuration>` are, see your desired configuration from [section: the actual build](#the-actual-build). Recall the note encourages using  `Release` and `x86`.
 
-> Note: If you want to build webrtc yourself, you will need to install Visual Studio 2015 with Update 3.
+<table>
+<tr>
+<th>Sample Type</th>
+<th>Sample Name</th>
+<th>Location</th>
+<th>Exe</th>
+</tr>
+<tr>
+<td>Server</td>
+<td>Spinning Cube</td>
+<td>SpinningCubeServer</td>
+<td>`SpinningCubeServer.exe`</td>
```

Comment:
we need to pull out these code tags, they don't work and it looks fine without them ☹️

Reply:
Agree